cli/pcluster/config/validators.py

```diff
@@ -531,6 +531,49 @@ def ec2_ebs_snapshot_validator(param_key, param_value, pcluster_config):
     return errors, warnings
 
 
+def ebs_settings_validator(param_key, param_value, pcluster_config):
+    """
+    Validate the following cases.
+
+    Number of EBS volume specified is lower than maximum supported
+    Parameter shared_dir is specified in every EBS section when using more than 1 volume
+    User is not specifying /NONE or NONE as shared_dir in EBS sections
+    """
+    errors = []
+    warnings = []
+
+    list_of_shared_dir = []
+    for section_label in param_value.split(","):
+        section = pcluster_config.get_section("ebs", section_label)
+        list_of_shared_dir.append(section.get_param_value("shared_dir"))
+
+    max_number_of_ebs_volumes = 5
+    num_volumes_specified = len(list_of_shared_dir)
+
+    if num_volumes_specified > max_number_of_ebs_volumes:
+        errors.append(
+            "Invalid number of EBS volumes ({0}) specified. Currently only supports upto {1} EBS volumes".format(
+                num_volumes_specified, max_number_of_ebs_volumes
+            )
+        )
+
+    if num_volumes_specified > 1 and None in list_of_shared_dir:
+        errors.append("When using more than 1 EBS volume, shared_dir is required under each EBS section")
+
+    return errors, warnings
+
+
+def shared_dir_validator(param_key, param_value, pcluster_config):
+    """Validate that user is not specifying /NONE or NONE as shared_dir for any filesystem."""
+    errors = []
+    warnings = []
+
+    if param_value == "/NONE" or param_value == "NONE":
```

Comment:
you can use a regex and print the param_value instead of a fixed string:
```python
if re.match("/?NONE"):
    errors.append("{0} cannot be used as a shared directory".format(param_value))
```

Reply:
In this case do we want to use `re.fullmatch("/?NONE", param_value)`? `re.match("/?NONE", param_value)` will also match strings starting with `/?NONE` like `/NONEshared`

Reply:
let's use `fullmatch` 👍

Reply:
`fullmatch` not compatible with Python 2.7, using `re.match("^/?NONE$", param_value)`